src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/discussions/PostsListPage.java

```diff
@@ -15,21 +17,24 @@
 
   @FindBy(css = ".post-detail")
   private List<WebElement> postList;
-
   @FindBy(css = ".sort")
   private WebElement sortEntryPointMobile;
-
   @FindBy(css = ".sort span")
   private WebElement labelInSortEntryPointMobile;
-
   @FindBy(css = ".discussion-sort")
   private WebElement sortOptionsMobile;
-
   @FindBy(css = "li[data-type='latest']")
   private WebElement latestLinkOnListMobile;
-
   @FindBy(css = "li[data-type='trending']")
   private WebElement trendingLinkOnListMobile;
+  @FindBy(css = ".back-button")
+  private WebElement backToWiki;
+  @FindBy(css = ".user-avatar")
+  private WebElement avatarImage;
+  @FindBy(css = ".avatar-username")
+  private WebElement avatarName;
+  @FindBy(css = "#WikiaUserPagesHeader")
+  private WebElement userPageHeader;
```

Comment:
please provide extra lines between each of `@FindBy..private` block, just as it was before